### DISCUSSION TITLE
Error Running rknn SSD Mobilenet Inference on RK3588 SOC using c++

### DISCUSSION THREAD

**Discussion by nirdesh17** [community/community#134840] [Programming Help]
I am encountering an error when running rknn ml model inference using C++ code for the SSD Mobilenet model on the RK3588 SOC. The error message is as follows:

```
E NPUTransfer: Cannot connect to proxy: Connection refused
E RKNNAPI: rknn_init, driver open fail! ret = -4(ERROR_NO_DEVICE)!
rknn_init fail! ret=-3
```
I am using the code from this repository: https://github.com/rockchip-linux/RKNPUTools/blob/rk33/mid/8.1/develop/rknn-api/Linux/rknn_api_sdk/rknn_ssd.cpp

Can anyone provide assistance with this issue?

**Comment by ebndev** [comment]
Hi @nirdesh17, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our `Programming Help` category, as this topic might be more relevant there and might help someone give you a nudge in the right direction 😁

**Reply by nirdesh17** [reply]
Hi @ebndev,

Thanks for moving my post to the `Programming Help` category! I appreciate your assistance.
